content/docs/pipelines/installation/standalone-deployment.md

```diff
@@ -206,28 +223,64 @@ bases:
 - github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref={{% pipelines/latest-version %}}
 ```
 
-### How to deploy, upgrade and uninstall using the repo
-Deploy: `kubectl apply -k $YOUR_REPO`
+### Use a repository to deploy, upgrade, and uninstall Kubeflow Pipelines
+
+**Note**: The following commands reference an existing repository (local or remote).
+
+* To deploy Kubeflow Pipelines using a repository, run:
+
+    ```
+    export KFP_MANIFESTS_REPO_LINK=<link-to-kfp-manifests-repository>
+    kubectl apply -k $KFP_MANIFESTS_REPO_LINK
+    ```
+
+    For example,
+
+    ```
+    export PIPELINE_VERSION=<kfp-version>
+    export KFP_MANIFESTS_REPO_LINK=github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+    kubectl apply -k $KFP_MANIFESTS_REPO_LINK
+    ```
+
+* To upgrade your Kubeflow Pipelines deployment from a repository, follow these steps:
 
-Upgrade:
+    **Caution**: It is recommended to back up your data storage for Kubeflow Pipelines.
 
-   1. (Recommended) Back up your data storages for KFP.
-   1. Edit `ref={{% pipelines/latest-version %}}` to a version you want to upgrade to.
+    1. Check [Kubeflow Pipelines GitHub repository](https://github.com/kubeflow/pipelines/releases) for available releases.
+    1. Set the `PIPELINE_VERSION` environment variable to a version you want to upgrade to.
+    1. Upgrade KFP by running:
 
-        Check [Kubeflow Pipelines github repo](https://github.com/kubeflow/pipelines/releases) for available releases.
-   1. Deploy: `kubectl apply -k $YOUR_REPO`.
+        ```
+        export PIPELINE_VERSION=<version-you-want-to-upgrade-to>
+        export KFP_MANIFESTS_REPO_LINK=github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+        kubectl apply -k $KFP_MANIFESTS_REPO_LINK
+        ```
 
-Uninstall: `kubectl delete -k $YOUR_REPO`.
+* To uninstall Kubeflow Pipelines using a repository, run:
 
-### Further reading
-* kustomize's [recommended workflow using an off-the-shelf configuration](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/workflows.md#off-the-shelf-configuration).
+    ```
+    export PIPELINE_VERSION=<kfp-version>
+    export KFP_MANIFESTS_REPO_LINK=github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+    kubectl delete -k $KFP_MANIFESTS_REPO_LINK
+    ```
 
 ## Troubleshooting
 
 ### Permission error installing Kubeflow Pipelines standalone to a cluster
 
-Run:
+If you encounter a permission error when installing Kubeflow Pipelines standalone, run:
 
 ```
-kubectl create clusterrolebinding your-binding --clusterrole=cluster-admin --user=[your-user-name]
+kubectl create clusterrolebinding your-binding --clusterrole=cluster-admin --user=<your-user-name>
 ```
+
+## Further reading
```

Comment:
I think these links are important in context. Consider moving these links back into the content of the page.

Reply:
OK